3_javascript/javascript1/week2/Flight booking fullname function.js

```diff
@@ -0,0 +1,16 @@
+function getFullname(firstname, surname, useFormalName, gender) {
+    if (useFormalName) {
+        if (gender === "male") {
+            return "Lord " + firstname + " " + surname;
+        } else if (gender === "female") {
+            return "Lady " + firstname + " " + surname;
+        }
+    }
+    return firstname + " " + surname;
+}
+const fullName = getFullname("Benjamin", "Hughes", true, "female");
+console.log(fullName);
+
```

Comment:
Nice thought adding the gender as a parameter of the function. Remember that you can use string literals for a cleaner code.